frontends/mit-open/src/page-components/SearchDisplay/SearchDisplay.tsx

```diff
@@ -392,10 +392,28 @@ const StyledDrawer = styled(Drawer)`
   }
 `
 
+const MobileFacetSearchButtons = styled.div`
+  width: 100%;
+  gap: 12px;
+`
+
+const MobileApplyFiltersButton = styled(Button)`
+  background-color: ${({ theme }) => theme.custom.colors.mitRed};
+  color: ${({ theme }) => theme.custom.colors.white};
+  padding: 12px;
+  border-radius: 4px;
+  width: 144px;
```

Comment:
In addition to my comment above... try to avoid explicit widths. Figma often specifies them because Steve and Bilal tend to design for a few specific screen sizes. Ideally,  try to match figma at full width, but keep roughly the same look and feel at other widths.

In particular, a fixed width here doesn't work great on some phones:

<img width="581" alt="Screenshot 2024-06-25 at 10 28 22 AM" src="https://github.com/mitodl/mit-open/assets/9010790/53b91bde-0e88-40f6-9ff7-345e10a9793a">
<img width="524" alt="Screenshot 2024-06-25 at 10 28 28 AM" src="https://github.com/mitodl/mit-open/assets/9010790/3adfc349-5951-445d-9aef-d31a9e00699a">

*Figma designs are for 375px screens, and the iphone SE is 375px, so you might be surprised the 144px didn't work well. I was surprised, too. I guess there's a slight deviation in Figma's drawer vs our implementation. Figma shows 300px content space. Our drawer has `max-width: 332px; padding-left/right: 16px, width: 85%`. The `width: 85%` is the difference, making the drawer a little smaller than 300px.*  Might be worth aligning that with Figma, but probably not here.

**Anyway:** A flex-based layout for the buttons would be a better way to match Figma at the 300px content width, but scale appropriately on slightly smaller phones.